book/hardware-batterybay.md

```diff
@@ -10,51 +10,44 @@ For information on interfacing with the battery, please visit the [Battery Inter
 
 To interface with Solo there are a few crucial interface points to pay attention to
 
-> <sub>* Interlocking with the battery tray
-
-> <sub>* Interfacing the electrical connection
-
-> <sub>* Using the release button
-
-> <sub>* Keep out zones for accessory payloads and props
+* Interlocking with the battery tray
+* Interfacing the electrical connection
+* Using the release button
+* Keep out zones for accessory payloads and props
 
 Most all of these interfaces can be found in the battery bay envelope cad file (link to come).
 
 Any battery options operating in the battery bay should minor approximately the same specs of the current battery:
 
-> <sub>* Weight: 500g
-
-> <sub>* Operating Temp: 0℃ ~ 40℃
-
-> <sub>* Discharging operating temp: -20℃ ~ 60℃
+* Weight: 500g
+* Operating Temp: 0C ~ 40C
+* Discharging operating temp: -20C ~ 60C
 
 ## Electrical Interface Specifications
 
 The electrical interface with Solo is done through the following connectors:
 
-> <sub>* Battery side female connector is a Molex 171090-0048 ([PDF](https://drive.google.com/open?id=0B9l93ZUM5ooxMU0xM3h5MzNsMjVBV3NjYU9DSEdyZE5FQWhR)). This is a custom connector that can only be purchased in minimum orders of 1,000 units.
+* Battery side female connector is a Molex 171090-0048 ([PDF](https://drive.google.com/open?id=0B9l93ZUM5ooxMU0xM3h5MzNsMjVBV3NjYU9DSEdyZE5FQWhR)). This is a custom connector that can only be purchased in minimum orders of 1,000 units.
 
-> <sub>* Solo / charger side male connector is a Molex 171088-0048 ([PDF](https://drive.google.com/open?id=0B9l93ZUM5ooxblFVOVhuY2JwMExjd3FnYjgzZmNsNy1ialAw))
+* Solo / charger side male connector is a Molex 171088-0048 ([PDF](https://drive.google.com/open?id=0B9l93ZUM5ooxblFVOVhuY2JwMExjd3FnYjgzZmNsNy1ialAw))
```

Comment:
This link is not shared publically on drive. Either share, move to another public location, or mark as "link to follow".